src/frontend/app/View/flowView/FlowView.js

```diff
@@ -13,7 +13,7 @@ export default class FlowView {
     this.moving = false;
     this.adding = false;
     this.connectorType = "Flowchart";
-    this.horizontalBuild = false;
+    this.horizontalBuild = true;
```

Comment:
I have to look at my configuration. There is a possibility that they saved the orientation wrong.